src/Tizen.Applications.Common/Tizen.Applications/EventSystem.cs

```diff
@@ -0,0 +1,172 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using static Interop.EventSystem;
+
+namespace Tizen.Applications
+{
+    /// <summary>
+    /// The EventSystem API provides functions to receive events.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class EventSystem : IDisposable
```

Comment:
I think the name "EventSystem" is too specific. Actually, this class does not about "system" it is just about "Event". How about renaming to Event?